packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -558,90 +558,109 @@ module.exports = async (api, args = {}, { pluginSource, activity } = {}) =>
       }
     }
 
-    Promise.mapSeries(implementingPlugins, plugin => {
-      if (stopQueuedApiRuns) {
-        return null
-      }
+    let apiRunPromiseOptions = {}
+    let runPromise
+    if (
+      process.env.GATSBY_EXPERIMENTAL_PARALLEL_SOURCING &&
+      api === `sourceNodes`
```

Comment:
Can we swap those? Accessing `process.env` can be expensive: https://github.com/nodejs/node/issues/3104 

```suggestion
      api === `sourceNodes` &&
      process.env.GATSBY_EXPERIMENTAL_PARALLEL_SOURCING
```

Reply:
wut? Who knew. Also so weird/unfortunate. Thanks!